http/server/api.go

```diff
@@ -50,6 +39,15 @@ func (s *rulesetAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		s.get(w, r, path)
 		return
 	case "POST":
+		if _, ok := r.URL.Query()["watch"]; ok {
+			if len(path) != 0 {
```

Comment:
I wonder if it might be nicer to have an explicit path to use to watch (e.g. `/watch-rulesets`), rather than a POST to `/rulesets`. Not sure.
:tulip:

Reply:
Actually, we need a POST anyway because we need to send a body in the request

Reply:
It was really the re-purposing of the endpoint for both purposes that felt odd to me. I don't mind much - we should really just be using gRPC and then we wouldn't need to care about such trivialities :)